XLS-0070d-credentials/README.md

```diff
@@ -112,11 +115,106 @@ It has a maximum length of 256 bytes, and cannot be an empty string.
 
-The `DepositPreauth` object tracks a preauthorization from one account to another. This object already exists on the XRPL, but is being extended as a part of this spec to also support credential preauthorization.
+## 3. Transaction: `CredentialCreate`
+
+This transaction creates a `Credential` object. It must be sent by the issuer.
 
 ### 3.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`TransactionType`|✔️|`string`|`UInt16`|The transaction type (`CredentialCreate`).|
+|`Account`|✔️|`string`|`AccountID`|The issuer of the credential.|
+|`Subject`|✔️|`string`|`AccountID`|The subject of the credential.|
+|`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+|`Expiration`| |`number`|`UInt32`|Optional credential expiration.|
+|`URI`| |`string`|`Blob`|Optional additional data about the credential (such as a link to the VC document).|
+
+### 3.2. Failure Conditions
+
+* The account in `Subject` doesn't exist.
+* The time in `Expiration` is in the past.
+* The `URI` field is empty or too long (limit 256 bytes).
+* The `CredentialType` field is empty or too long (limit 256 bytes).
+* The account doesn't have enough reserve for the object.
+* A duplicate credential already exists (with the same `Subject`, `Issuer`, and `CredentialType`).
+
+### 3.3. State Changes
+
+If the transaction is successful:
+* The `Credential` object is created.
+* If `Subject` === `Account` (i.e. the subject and issuer are the same account), then the `lsfAccepted` flag is enabled.
+
+## 4. Transaction: `CredentialAccept`
+
+This transaction accepts a credential issued to the `Account` (i.e. the `Account` is the `Subject` of the `Credential` object). The credential is not considered valid until it has been transferred/accepted.
+
+### 4.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`TransactionType`|✔️|`string`|`UInt16`|The transaction type (`CredentialAccept`).|
+|`Account`|✔️|`string`|`AccountID`|The subject of the credential.|
+|`Issuer`|✔️|`string`|`AccountID`|The issuer of the credential.|
+|`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+
+### 4.2. Failure Conditions
+
+* The account in `Issuer` doesn't exist.
+* There is no valid credential described by the fields of the transaction.
+* The credential has already been accepted.
+* The `Account` doesn't have enough reserve for the object.
+* The credential has already expired.
+
+### 4.3. State Changes
+
+If the transaction is successful:
+* The `lsfAccepted` flag is turned on in the credential.
+* The burden of reserve for the `Credential` object is moved from the issuer to the subject.
```

Comment:
This should be removed then

Reply:
No, that need is still there. The reserve transfers, but there's only ever one.